live/tests/integration/components/snapshot-list-test.js

```diff
@@ -0,0 +1,78 @@
+import { expect } from 'chai';
+import RSVP from 'rsvp';
+import { describe, it } from 'mocha';
+import { setupComponentTest } from 'ember-mocha';
+import wait from 'ember-test-helpers/wait';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+
+describe('Integration | Component | snapshot list', function() {
+  setupComponentTest('snapshot-list', {
+    integration: true
+  });
+
+  it('renders', function() {
+    const organization = Ember.Object.create({ id: 1, snapshots: RSVP.resolve([]) });
+    this.set('organization', organization);
+
+    this.render(hbs`{{snapshot-list organization=organization}}`);
+    expect(this.$()).to.have.length(1);
+  });
+
+  it('should inform the user when no profile', function() {
+    // Given
+    const organization = Ember.Object.create({ id: 1, snapshots: RSVP.resolve([]) });
+    this.set('organization', organization);
+
+    // When
+    this.render(hbs`{{snapshot-list organization=organization}}`);
+
+    // Then
+    expect(this.$('.snapshot-list__no-profile')).to.have.length(1);
+    expect(this.$('.snapshot-list__no-profile').text()).to.equal('Aucun profil partagé pour le moment');
+  });
+
+  it('should display two snapshot items if two snapshots items shared', function() {
```

Comment:
Je propose un truc du genre : it should display as snapshot items as shared

Reply:
it should display **as many** snapshot items as shared